doc/pages/expansions.asciidoc

```diff
@@ -3,193 +3,192 @@
 == Strings
 
 \'strings'::
-    uninterpreted strings, use a backslash (\') to escape the separator
+    uninterpreted strings, use a backslash (\') to escape the separator.
 
 "strings"::
-    expanded strings, % strings (c.f. next section) contained are expended,
-    use a backslash (\\%) to escape the separator
+    expanded strings, `%` strings (c.f. next section) contained are expended,
+    use a backslash (\\%) to escape the separator.
```

Comment:
Thats actually out of date, % is escaped by doubling-up

Reply:
I left it there for the same reason as the above reply, there's a bug with backslash characters.

I'll change both bits.